easybuild/easyconfigs/n/netCDF/netCDF-4.3.3.1-intel-2015a.eb

```diff
@@ -0,0 +1,32 @@
+name = 'netCDF'
+version = '4.3.3.1'
+
+homepage = 'http://www.unidata.ucar.edu/software/netcdf/'
+description = """NetCDF (network Common Data Form) is a set of software libraries 
+ and machine-independent data formats that support the creation, access, and sharing of array-oriented 
+ scientific data."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = [
+    'ftp://ftp.unidata.ucar.edu/pub/netcdf/',
+    'ftp://ftp.unidata.ucar.edu/pub/netcdf/old',
+]
+
+dependencies = [('HDF5', '1.8.15')]
+
+builddependencies = [
+    ('CMake', '3.2.2'),
+    ('Doxygen', '1.8.9.1', '', ('GCC', '4.9.2')),
+    ('cURL', '7.43.0'),
+]
+
+# make sure both static and shared libs are built
+configopts = [
+    "-DCURL_LIBRARY=$EBROOTCURL/lib/libcurl.so -DCURL_INCLUDE_DIR=$EBROOTCURL/include -DBUILD_SHARED_LIBS=ON -DSZIP_LIBRARY=$EBROOTSZIP/lib/libsz.so",
+    "-DCURL_LIBRARY=$EBROOTCURL/lib/libcurl.so -DCURL_INCLUDE_DIR=$EBROOTCURL/include -DBUILD_SHARED_LIBS=OFF -DSZIP_LIBRARY=$EBROOTSZIP/lib/libsz.a",
```

Comment:
@boege: Well, the interesting thing was that the original easyconfig (without the -DSZIP_.... option) worked in 2 out of 3 machines. On the 3rd machine, the Szip was not found by cmake automatically, so this is why I had to insert that part. The test that you uploaded only a pertial log, so I am not able to tell, what was there a problem, because I do not have the whole log. If I want to investigate that, I have to see, whether it is really problem of pthereads, or it was also a problem with Szip (or something else).